lambdatimes/src/components/Carousel/Carousel.js

```diff
@@ -1,35 +1,52 @@
-import React, { Component } from 'react';
+import React, { Component } from 'react'
```

Comment:
Your carousel looks really good. Here's a link to slick, in case you might want to give it a try in the future. https://kenwheeler.github.io/slick/